assets/javascripts/job_next_previous.js

```diff
@@ -86,7 +86,11 @@ function renderJobResults(data, type, row) {
         if (count++) {
             var more = row.failedmodules.length - count + 1;
             if (more > 0 && limit < 12) {
-                html += '+' + more;
+                html += '<a title="';
+                for (var j = i; j< row.failedmodules.length ; j++) {
+                    html += "- " + row.failedmodules[j] + "\n";
```

Comment:
Shouldn't this use `htmlEscape`? (And maybe the surrounding code not touched by you also should.)

Reply:
And maybe you can use a `<ul><li>...</li></ul>` list and instead of the loop join by `</li><li>`.

Reply:
Done at least for the module name.

Reply:
The `<ul><li>` stuf doesn't work within `title` attribute.

Reply:
Ok, thought Bootstrap would allow that.